sky/provision/gcp/instance_utils.py

```diff
@@ -209,6 +247,52 @@ def delete_firewall_rule(
             firewall=firewall_rule_name,
         ).execute()
 
+    @classmethod
+    def get_vpc_name(
+        cls,
+        project_id: str,
+        zone: str,
+        instance_name: str,
```

Comment:
Should we use the same interface as other functions, i.e. `instance` instead of `instance_name`?